internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -34,6 +35,12 @@ type Pipeline struct {
 	Name string `json:"name"`
 }
 
+// PipelineStatus represents a Pipeline's status.
+type PipelineState struct {
+	PipelineName string `json:"Pipeline"`
+	StageStates []*cp.StageState `json:"StageStates"`
```

Comment:
So if the JSON key isn't different from the struct key, you don't need to specify the json string since that's the default behavior when the struct is marshalled. However I think we've been keeping this lower case so like above, `json:"stageStates"`